### ISSUE TITLE
Feature Request: Feedback system

### ISSUE THREAD

**Issue by panzerscope** [netwaretcs/Classifieds#14]
The classifieds plugin is almost perfect and a great resource for the invision forums, but It is missing one element, Feedback ratings. 

It would be great if buyers, sellers & traders were able to leave feedback on a classified. This helps to show who within any given community has a good history in the classifieds section and will help weed out the trust worthy from the flaky and all in-between. 

VBulletin once upon a time had the good old I-trader system which worked really well, but nobody seems to have replicated its success on Invision, or many other platforms for that matter. 

I think the ability to leave feedback would be a much welcomed addition for many looking to buy the plugin and for current customers. 

Many Thanks,
P

**Comment by netwaretcs** [comment]
Yes, it is one of the major features I plan to implement on Classifieds. 
Every time I've started working on it I had doubts about how to proceed.
 It would be helpful to have some concrete examples, like what exactly this i-trader do.

**Comment by numis-ips** [comment]
Agree. That's one of thing that I planned to ask you when we had conversation. But same as you I didn't have enough time to think over the final product. I'll try to make an example in the near future as another enhancements too.

**Comment by panzerscope** [comment]
> Yes, it is one of the major features I plan to implement on Classifieds.
> Every time I've started working on it I had doubts about how to proceed.
> It would be helpful to have some concrete examples, like what exactly this i-trader do.

I am going to send you a private message with documentation on how I-Trader works to help you with this :)

**Comment by jmillerinvision** [comment]
Love this idea 👏

**Comment by numis-ips** [comment]
Finally, there was a little time to think about the system. It is necessary for users to intuitively separate the product rating (block Reviews) and the seller rating (which needs to be created) from each other.

Reviews:
It must be remembered that the goods sold can be mass (electronics, handmade things) - a rating is needed for such.
So are collectibles (single) - the rating is not very useful, but its presence does not interfere with anything.
The rating under the product is now located where it is needed and it seems to me there is no need to change it:
![image](https://user-images.githubusercontent.com/83723982/121185737-0b36d500-c86f-11eb-83b5-6b58c324f772.png)
Reviews, in this case, I would rename them to product reviews. After all, first of all, we put a five-star rating on the product and already in the text, users will describe their opinion about the product, and someone else the deal itself - it is necessary to somehow intuitively show them that the product is being evaluated here, and not the deal itself. 
![image](https://user-images.githubusercontent.com/83723982/121186097-65379a80-c86f-11eb-83e9-c6c806bbf468.png)
Ultimately, we needed a system where people could leave an overall rating for a product, form a simple rating (5-star) and leave their opinions about it. Such a system already exists. It will also be superfluous to combine the rating of the same products from different sellers. I think this is too difficult. 
Now let's think about the reputation of the sellers.

**Comment by numis-ips** [comment]
Unfortunately I don't have free time right now to fully think about the feedback system and the addon in general.
Similarly, I delayed the launch of my forum. I will definitely come back to this question later. 

For those who have the opportunity, I would advise you to look at the russian site auction.ru with chrome translation.
It is 90% similar to what we do. It is a leader platform with huge traffic and well-thought-out design. So I'm sure that all the answers are there.
https://auction.ru/offer/as_1_kopejka_1756_bb_orel_v_oblakakh_gurt_setka_r1_zheltaja_053-i188585899872083.html
![image](https://user-images.githubusercontent.com/83723982/123544829-d5925700-d75d-11eb-9c22-87fb9e09972b.png)
